Upstream build process doesn't set an explicit version of Python because script
works for Python 2.x and 3.x.  But Solaris requires an explicit one, so prefer
3.4.

configure should detect Python version and allow builder to override
detection via --with-python or a PYTHON environment variable, and then the
build process should replace the first line of the script instead.

XXX file upstream bug

--- gupnp-1.0.2/tools/gupnp-binding-tool	2015-03-30 10:49:17.000000000 -0700
+++ gupnp-1.0.2/tools/gupnp-binding-tool	2015-07-05 21:13:08.428030160 -0700
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3.5
 
 # Copyright (C) 2008 OpenedHand Ltd
 # Copyright (C) 2008 Intel Corporation
